### PR TITLE
Improve "Creating symbol packages" documentation

### PR DESCRIPTION
**Pull request by loic-sharma**
* Link to Source Link for source debugging
* Other doc tweaks

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[e2eac76](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e2eac7686b2132be338f409867300dd572374fb5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1675) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909102050462928-1675%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=nGoF%2BGT36IN%2FGejymhKDpXXKVhS6Y9Ptl2Q%2BLOPd0T4%3D&st=2019-09-10T20%3A46%3A51Z&se=2019-10-11T20%3A51%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[d1afdb3](https://github.com/NuGet/docs.microsoft.com-nuget/commits/d1afdb35a4bf01172d3fced75c4dfe5f2c68589c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1675) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909102051485981-1675%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=YKEXrht%2B3mqWuDvz75IliF27z1CMt1rtJNS7epoEwgY%3D&st=2019-09-10T20%3A48%3A03Z&se=2019-10-11T20%3A53%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[b08959e](https://github.com/NuGet/docs.microsoft.com-nuget/commits/b08959ec0ecd9924891b6d63faa5efb723303fa6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1675) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909102053101892-1675%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=V7bSPsDjUDXg6GZfTvLJTtmcMcw3Ckb3JOaIo6RXPec%3D&st=2019-09-10T20%3A49%3A10Z&se=2019-10-11T20%3A54%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[2a84d56](https://github.com/NuGet/docs.microsoft.com-nuget/commits/2a84d56fd378dfcd62ed4a8901a67fcc9997bf85)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1675) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909102053539293-1675%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=EUkRcdo186EznAEQMlmkU26ZayuGf8omKTT%2FBLqR8ws%3D&st=2019-09-10T20%3A50%3A23Z&se=2019-10-11T20%3A55%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[15fccd4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/15fccd46af53d02a3b623640d8a3b6d0fe8851db)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Symbol-Packages-snupkg.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loic-sharma-patch-2/docs/create-packages/Symbol-Packages-snupkg.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg?branch=pr-en-us-1675) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201909102113160554-1675%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hDV1dJFE1rmco5UfL%2F0f91c%2F29zhJL30InY1Ocr5cZ0%3D&st=2019-09-10T21%3A09%3A13Z&se=2019-10-11T21%3A14%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Symbol-Packages-snupkg.md [hunk 1]
```diff
@@ -128,4 +127,6 @@ The .nupkg file would be exactly the same as it is today, but the .snupkg file w
 
 ## See Also
 
-[NuGet Package Debugging & Symbols Improvements](https://github.com/NuGet/Home/wiki/NuGet-Package-Debugging-&-Symbols-Improvements)
+Consider using Source Link to enable source code debugging of .NET assemblies. For more information, please refer to the [Source Link guidance](/dotnet/standard/library-guidance/sourcelink.md).
```
Comment (shishirx34) [inline docs/create-packages/Symbol-Packages-snupkg.md]: Is the sourcelink URL correctly configured for relative hit?

Comment (shishirx34) [reply docs/create-packages/Symbol-Packages-snupkg.md]: may be use absolute url?

Comment (loic-sharma) [reply docs/create-packages/Symbol-Packages-snupkg.md]: Talked offline, this is the pattern used to reference a docs page. We can't test the URL is correct until after we've merged sadly :/

**Pull request merged**